Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/playwright:v1.45.1-jammy
+
+COPY . .
+
+RUN npm install
+RUN npx playwright install
+
+EXPOSE 9329
+
+ENTRYPOINT [ "npx", "playwright", "test" ]
```

Comment:
This would be better to execute a script command that is defined in the `package.json` file

Reply:
any suggestions for that, while keeping it flexible? I agree that it's good practice, but with this way above we can pass the ENV for which environment we want when we run the image (e.g. docker run -e NODE_ENV=live ...)

Reply:
made changes in upcoming commit